crates/adkg-cli/README.md

```diff
@@ -105,8 +105,44 @@ adkg-cli run                                \
   --priv adkg.priv                          \
   --id 1                                    \
   --listen-address "/ip4/0.0.0.0/tcp/7777"  \
+  --transcript-out adkg.transcript          \
   --priv-out adkg.ceremony.priv             \
   --pub-out adkg.ceremony.pub 
 ```
 
 Notice that we also include two output files used to store the private and public output of the ADKG.
+
+We also include a third file, with `--transcript-out`, which is used to store an encrypted transcript of messages sent during the ADKG.
+The transcript should be sent to the other participants should a node fail to run the `ADKG` ceremony.
+
+### Additional options
+**Logging**\
+The log level can be changed using the `LOG_LEVEL` env variable.
+The specified level should follow the [`EnvFilter`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives) syntax.
+
+**Metrics**\
+A prometheus metrics endpoint may be enabled by adding the `--features metrics` flag when compiling the cli, and by adding the `--metrics` flag to the `run` command.
+This exports metrics on the `127.0.0.1:8080/metrics` endpoint by default.
+
+## Rescue ADKG
```

Comment:
could you link the paper/relevant section here? I tried to find it to follow along with the code but couldn't

Reply:
Understood - discussed this with @AnomalRoil and it makes sense

Reply:
I think the recovery mechanism would make a good blog post!